packages/manager/modules/telecom-universe-components/src/telecom/pack/migration/pack-migration-process.service.js

```diff
@@ -368,6 +370,18 @@ export default /* @ngInject */ function($q, OvhApiPackXdsl, Poller) {
     migrationProcess.buildings = buildings;
   };
 
+  /*= ===============================
+    =          Meeting              =
+    ================================= */
+  self.setSelectedMeeting = function setSelectedMeeting(
+    meetingSlots,
+    contactName,
+  ) {
+    migrationProcess.selectedOffer.contactName = contactName;
+    migrationProcess.selectedOffer.meetingSlots = meetingSlots;
+    migrationProcess.currentStep = 'confirm';
```

Comment:
Can you move 'confirm' to constant file. We can reuse same in other places as well.

Reply:
On this service, the value is used one time